package.json

```diff
@@ -98,5 +99,15 @@
     "**/*.{js,jsx,ts,tsx,json,css,scss,md}": [
       "prettier --write"
     ]
+  },
+  "jest": {
+    "collectCoverageFrom": [
```

Comment:
Can we exclude the build folder too? We don't have that one (/build) on the repo, but do have it on our local dev env.

Reply:
Added the line to exclude builder folder, but I think if you run the command, the test does not include the build folder by itself unless you are talking about something else?

Reply:
correct me if im wrong, I think only files in the `src` directory is included based on the first line. So I don't think excluding the build and node_modules directories is needed.

Reply:
I think you are right. I usually leave it there just in case and because the documentation had it in the example :sweat_smile: